### DISCUSSION TITLE
Release quarto CLI 1.4.* ?

### DISCUSSION THREAD

**Discussion by th0ger** [quarto-dev/quarto-cli#7798] [Q&A]
Hey Quarto devs. Thanks for the amazing product.

It it possible to ship a new version of Quarto CLI?
Latest CLI release is 1.3.450 / Aug. 4 2023 https://quarto.org/docs/download/release.html

This release is shipping an older version of Mermaid 9.1.1 with bugs (specifically for me https://github.com/mermaid-js/mermaid/pull/3726, fixed in https://github.com/mermaid-js/mermaid/releases/tag/v9.3.0)

When I run the docker image `ghcr.io/quarto-dev/quarto:1.4.518` it's working fine.

**Comment by mcanouil** [comment]
Quarto will always ship "outdated" third parties especially when third parties release cycle is 1 to 3 times a month.
Quarto cannot and will not align its releases on third parties, because currently that would mean at least: Bootstrap/Bootswatch, mermaid, observable.js, Pandoc, etc., thus the actual nearly daily pre-releases built would be the actual releases which you can already use as you pointed out.

For more:
- #7765

Closing this as duplicate.

**Reply by th0ger** [reply]
Yes. I fully understand that Quarto can't and should not have its release schedule mandated by externals.

We use both the Docker release for local and production builds. But we also use the CLI for local development (together with VS Code extension). And we don't have an option to install the nightly builds for CLI.

I don't know your exact release/patch strategy for CLI. But: Given the long release cycle for quarto official releases, wouldn't it make sense to either:

- Provide a nightly CLI build, based on the same code as docker builds.
- Release monthly or quarterly patch releases of quarto CLI that picks up all patches/bugfixes from depending libraries?

**Reply by cscheid** [reply]
> Provide a nightly CLI build

You mean [these](https://github.com/quarto-dev/quarto-cli/releases)?

**Reply by mcanouil** [reply]
Releases and pre-releases can be downloaded from GitHub releases assets as linked by Carlos or from Quarto documentation website: https://quarto.org/docs/download/

**Reply by th0ger** [reply]
Perfect!
How did I miss those. Sorry for rambling.
